app/controllers/miq_policy_controller/policies.rb

```diff
@@ -241,8 +241,8 @@ def policy_build_edit_screen(edit_type = nil)
       MiqPolicy.all_policy_events.each do |e|
         next if excluded_event?(e)
 
-        @edit[:allevents][e.etype.description] ||= []
-        @edit[:allevents][e.etype.description].push([e.description, e.id.to_s])
+        @edit[:allevents][_(e.etype.description)] ||= []
+        @edit[:allevents][_(e.etype.description)].push([_(e.description), e.id.to_s])
```

Comment:
do we really need the translated value as the hash key?  I would think the only place we need the translation is inside the push - which is probably what is being displayed